packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/desktop/my-jobs-table.tsx

```diff
@@ -203,9 +206,13 @@ const getColumnsDefinition = ({
               </TableButton>
               <RejectButton
                 disabled={buttonDisabled}
+                loading={isRejectPending}
                 onClick={() => {
                   if (buttonDisabled) return;
```

Comment:
Not sure why we need this `if` since if button is "disables" - it can't be clicked, but it was there.
Could you please clean it up?

Reply:
It's good that you noticed it, because thanks to it I noticed that there is a separate table for mobile and I also implemented loading state there